index.html

```diff
@@ -36,35 +33,191 @@ <h1>
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+        <div class="work multipost">
+          <img class="work-img" src="./image/image-work4.png" />
+          <h2 class="work-title">Multi-Post Stories</h2>
+          <ul class="information">
+            <li>CANOPY</li>
+            <img src="./image/point.png" />
+            <li>Back End Dev</li>
+            <img src="./image/point.png" />
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </section>
+      <section id="section3">
+        <h2>
+          About
+          <br />
+          Myself
+        </h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <h3>LET'S CONNECT</h4>
+        <div id="icons">
+          <ul id="list-icons">
+            <li class="img">
+              <a href="#">
+                <img src="./image/tweeter.png" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/Linkedin icon.jpg" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/m.png" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/github.png" alt="" />
+              </a>
+            </li>
+            <li class="img">
+              <a href="#">
+                <img src="./image/hand.png" alt="" />
+              </a>
+            </li>
+          </ul>
+        </div>
+        <button>Get my resume</button>
+        <div class="categories">
+          <h4>Languages</h4>
+          <img src="./image/arrow-down.png" alt="" />
+        </div>
+        <ul class="dropdown-menu">
+          <li>
+            <img src="./image/Ellipse1.png" />
+            JavaScript
+          </li>
+          <li>
+            <img src="./image/Ellipse2.png" />
+            HTML
+          </li>
+          <li>
+            <img src="./image/Ellipse3.png" />
+            CSS
+          </li>
+        </ul>
+        <div class="categories">
+          <h4>Framekors</h4>
+          <img src="./image/Union.png"/>  
+        </div>
+        <div class="categories">
+          <h4>Skills</h4>
+          <img src="./image/Union.png"/>
+        </div>

```

Comment:
- [ ] For the Languages section we have to make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` also the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`, like :point_down: 
```
 <ul>
      <li> Language</li>
        <ul>
          <li>Javascript</li>
          <li>Ruby</li>
          <li>Html</li>
          <li>Css</li>
        </ul>
```